modin/pandas/test/dataframe/test_default.py

```diff
@@ -762,6 +762,137 @@ def test_resample_specific(rule, closed, label, on, level):
         )
 
 
+@pytest.mark.parametrize(
```

Comment:
Please, reduce amount of code using the next:
1) Use existing approaches to generate DateTime-like indexes
2) Combine both tests to one
3) Use `eval_general` to check equality of pandas/modin results

Reply:
This was changed